updating docs service -> cloud (#2666)
pulumi/pulumi-hugo: themes/default/content/docs/guides/continuous-delivery/add-support-for-cicd-systems.md

- PR Number
- Commit Message

> The above metadata about your CI environment is then used to provide an enhanced experience in the [Pulumi Cloud](https://app.pulumi.com) when you look at your stack activity log. The metadata from your CI environment combined with the information about your Git repository allows us to provide links to things such as the Pull Request on github.com or the commit that triggered a PR build.

Author: ```suggestion
The metadata about your CI environment is displayed in the [Pulumi Cloud](https://app.pulumi.com) stack activity log. The metadata from your CI environment and the information about your Git repository allow Pulumi to provide links to pull requests and commits.
```

### Detecting Additional Metadata About A CI Build

> A CI build could have been triggered by a PR or a push build. In the case of the former, the linked PR number is useful so that the Pulumi Cloud can add a link to it in the source repo of your stack. Note that the detection is done on a best-effort basis. We support linking to the following Git-based version control systems:

Author: ```suggestion
A CI build could have been triggered by a PR or a push build. In this case, the linked PR number is useful so that the Pulumi Cloud can link to it. Note that the detection is done on a best-effort basis. We support linking to the following Git-based version control systems:
```
